parm/soca/letkf/letkf.yaml.j2

```diff
@@ -38,32 +41,31 @@ local ensemble DA:
     mult: 1.1
 
 output:
-  datadir: data_output/
+  datadir: Data/
```

Comment:
Wouldn't a more descriptive output path be useful than `Data/` for all of these?

Reply:
As I recall this was to match existing code but I don't see a reason why it can't be changed

Reply:
`Data` sounds good to me, but up to you @AndrewEichmann-NOAA .

Reply:
Chaanged to letkf_output